.rubocop.yml

```diff
@@ -0,0 +1,84 @@
+AllCops:
+  Include:
+    - '**/Rakefile'
+  Exclude:
+    - 'bin/stubs/*'
+    - 'cookbooks/**/*'
```

Comment:
This file should now be not have any ruby in it anymore. It can be removed from here